temporalio/lib/temporalio/client.rb

```diff
@@ -334,6 +337,106 @@ def workflow_handle(
       WorkflowHandle.new(client: self, id: workflow_id, run_id:, result_run_id: run_id, first_execution_run_id:)
     end
 
+    # Start an update, possibly starting the workflow at the same time if it doesn't exist (depending upon ID conflict
+    # policy). Note that in some cases this may fail but the workflow will still be started, and the handle can then be
```

Comment:
`this` is unclear in this sentence.

```suggestion
    # policy). Note that in some cases, the Update message may fail though the Workflow was sucessfully started; the handle can then be
```